app/TBG.py

```diff
@@ -112,12 +113,15 @@ def create_new_game() -> Dict:
         player: Player = Player(request.json['name'])
     except KeyError:
         abort(400, util.error('Name not supplied'))
-    
-    public: bool = False
-    if 'public' in request.json:
-        public = request.json['public']
 
-    game: Game = Game(public)
+    try:
+        if not request.json['game_type'] in GAME_TYPES:
+            abort(400, util.error('Invalid game type parameter'))
```

Comment:
Having an abort inside a `try` block seems.. odd. It'd be cleaner if you only had the try block check if the value is in the post data.

Example:
```
try:
    game_type=request.json['game_type']
except KeyError:
    abort(...)
if game_type not in Game.GAME_TYPES:
    abort(...)
```

Reply:
Update: this still holds true. Just change Game.GAME_TYPES to `('public', 'private')`